cmd/operator/deploy/rule-evaluator/02-rolebinding.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: gmp-system
```

Comment:
The standalone rule-evaluator deployment actually doesn't specify a namespace to be deployed in.

This is intentional as we don't want to prescribe a particular namespace one has to install the rule-evaluator into. This is in contrast to the managed-collection stack, which straddles two namespaces (a system and public namespace, default `gmp-system` and `gmp-public`, respectively).

I would rather see a dedicated service account for the rule-evaluator being created here, i.e. `rule-evaluator`, that would be added to the deployment spec, and that these `Role` and `RoleBinding`s used against.

Can you add that and split them out? You can follow the [pattern](https://github.com/GoogleCloudPlatform/prometheus-engine/tree/f052681d5c47d9dba0d2e9e596ee52e11c856e1a/cmd/operator/deploy/operator) used for the operator and its [service account](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/f052681d5c47d9dba0d2e9e596ee52e11c856e1a/cmd/operator/deploy/operator/05-deployment.yaml#L54).

Reply:
Thank you for your review!
I added the new serviceAccount `rule-evaluator`.